### ISSUE TITLE
Mortar is sometimes unable to deploy without explanation

### ISSUE THREAD

**Issue by dynamittt** [DarklightGames/DarkestHour#536]
5.1 bug, spawn and immediately attempt to sprint while holding the mortar. Then attempt to deploy mortar-nothing happens. No message shown.
Will not be able to deploy at all, so only thing left to do is respawn.

**Comment by dynamittt** [comment]
Just to let you know, I had a similar issue with nades in one of my mutators, in this case it was caused by LoweringWeapon state being invoked from within the Pawn, at the same time another function asked for the weapon to switch state, may be something similar is happening here? May be, if you attempt sprinting just while weapon is being "BringedUp" right after respawn, something gets screwd up.

**Comment by handsm** [comment]
I don't think is a problem any more. The mortar code has been re-worked in several areas & it's possible one of many changes has fixed this, although wasn't targeted at it.

The given reproduction steps are quite specific. I have just tried them on a dedicated server, numerous times, but can no longer reproduce the bug. I spawned & immediately began sprinting. My player held on to his gun & didn't bring up the mortar until I stopped. I deployed immediately on stopping stringing & had no problem doing this.
